design/workitemtype.go

```diff
@@ -111,52 +111,41 @@ var workItemTypeSingle = JSONSingle(
 	workItemTypeData,
 	workItemTypeLinks)
 
-var _ = a.Resource("workitemtype", func() {
-	a.Parent("space")
+var _ = a.Resource("workitemtypes", func() {
+	a.Parent("space_template")
```

Comment:
I see that you have added `a.Parent("space_template")` for `list` action of `workitemtypes` and `work_item_link_types` but not for the `list` action of `space`. Is there a reason for it?

Reply:
Yes, there's a reason, please look at the model: http://www.plantuml.com/plantuml/proxy?idx=0&src=https://raw.githubusercontent.com/kwk/fabric8-wit/spacetemplate/spacetemplate/space_templates.puml&fmt=svg . I don't see a reason why we want to list all spaces that implement a template.